instrumentation/action_pack/test/opentelemetry/instrumentation/action_pack/handlers/action_controller_test.rb

```diff
@@ -65,7 +64,7 @@
 
     _(last_response.body).must_equal 'created new item'
     _(last_response.ok?).must_equal true
-    _(span.name).must_equal 'ExampleController#new_item'
+    # _(span.name).must_equal 'GET example/new_item'
```

Comment:
Should these be updated or removed?

Reply:
Oh my. That definitely was a mistake on my part.

Thank you for pointing this out!